src/sentry/integrations/pagerduty/integration.py

```diff
@@ -134,7 +134,7 @@ class PagerDutyIntegrationProvider(IntegrationProvider):
     key = "pagerduty"
     name = "PagerDuty"
     metadata = metadata
-    features = frozenset([IntegrationFeatures.ALERT_RULE])
+    features = frozenset([IntegrationFeatures.ALERT_RULE, IntegrationFeatures.INCIDENT_MANAGEMENT])
```

Comment:
@MeredithAnya The feature descriptions should be updated as well to match

Reply:
PagerDuty already had the feature description for incident management